pkg/audit/manager.go

```diff
@@ -155,6 +155,14 @@ func (am *Manager) audit(ctx context.Context) error {
 		return nil
 	}
 
+	// get all constraint kinds
+	rs, err := am.getAllConstraintKinds()
+	if err != nil {
```

Comment:
should we also check the length of rs to see if tis empty?

Reply:
`ServerResourcesForGroupVersion` inside `getAllConstraintKinds` returns error when no `getAllConstraintKinds` is found in cluster so rs is not initialized

Reply:
ack